netbout-web/pom.xml

```diff
@@ -733,6 +733,7 @@
                         <artifactId>qulice-maven-plugin</artifactId>
                         <configuration>
                             <excludes>
+                                <exclude>checkstyle:/src/main/java/com/netbout/rest/TkStart.java</exclude>
```

Comment:
@dmzaytsev The suggestion by @carlosmiranda was for using `Pair` object from apache commons lang-3 (see again). Unfortunately, the exclusion isn't something we can presently remove (See [qulice issue 608](https://github.com/teamed/qulice/issues/608) raised for the same).